test/fail_compilation/traits.d

```diff
@@ -0,0 +1,24 @@
+/*
+REQUIRED_ARGS:
+PERMUTE_ARGS:
+*/
+
+/************************************************************/
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/traits.d(100): Error: `getTargetInfo` key `"not_a_target_info"` not supported by this implementation
+fail_compilation/traits.d(101): Error: string expected as argument of __traits `getTargetInfo` instead of `100`
+fail_compilation/traits.d(102): Error: expected 1 arguments for `getTargetInfo` but had 2
```

Comment:
Very minor nit. Most error messages use found, so:
"string expected as argument of __traits `getTargetInfo`, found `100`"
"expected 1 arguments for `getTargetInfo`, found 2"

Reply:
As usual, I copy&pasted the error message from the ones nearby to maintain the same language as the surrounding precedent.
The error message about the argument count is emit by a utility function that produces the same error for EVERY `__traits`.

I don't know how many more times I can be told to change code that I copied from a few lines away ;)

Reply:
Haha.

>I don't know how many more times I can be told to change code that I copied from a few lines away ;)

Lots ;) Don't worry about it.